app/genesis.go

```diff
@@ -4,15 +4,19 @@ import (
 	"github.com/tendermint/tendermint/crypto"
 )
 
+// GenesisState specifies the validator of the chain
 type GenesisState struct {
 	Validator GenesisValidator `json:"validator"`
 }
 
+// GenesisValidator holds the consensus public key and fee address of
```

Comment:
Should note that ConsPubKey is tendermint `Ed25119` keys (check this first i forget exact sig algorithm we use).
While the Fee Address is an Ethereum address